mbtools/generate_mbz_toc.py

```diff
@@ -48,17 +49,67 @@ def create_toc(mbz_path, md_filepath):
                             current_page = id2page[current_page.next]
 
 
+def create_toc_csv(mbz_path, csv_filepath):
```

Comment:
High-level comment: This seems to work, but it seems a shame that there is so much duplicity between `create_toc_md` and `create_toc_csv`. Perhaps a potential refactor would be to have both rely on a separate method that builds the toc tree as an object (perhaps `parse_toc` or something), and then each simply converts the tree to the specific output :man_shrugging: . Or, maybe `parse_toc` gets called in `main()` and passes the object to the appropriate serialization function depending on `--csv` :man_shrugging:.

Thoughts?